pipeline/subject.py

```diff
@@ -0,0 +1,53 @@
+'''
+Schema of subject information.
+'''
+import datajoint as dj
+from pipeline import reference
+
+schema = dj.schema('gao2018_subject')
+
+
+@schema
+class Species(dj.Lookup):
+    definition = """
+    species_name: varchar(64)
+    """
+    contents = [['Mus musculus']]
+
+@schema
+class Strain(dj.Lookup):
+    definition = """
+    strain_id: varchar(64)
```

Comment:
let's rename `strain_id` to `strain`, add a comment, and make the datatype shorter, e.g. `varchar(24)`